flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -18,15 +18,14 @@
 from flytekit.common.exceptions.scopes import system_entry_point
 from flytekit.common.tasks import task as _sdk_task
 from flytekit.common.utils import write_proto_to_file as _write_proto_to_file
-from flytekit.configuration import auth as _auth_config
 from flytekit.configuration import internal as _internal_config
 from flytekit.tools.fast_registration import compute_digest as _compute_digest
 from flytekit.tools.fast_registration import filter_tar_file_fn as _filter_tar_file_fn
 from flytekit.tools.module_loader import iterate_registerable_entities_in_order
 
-_PROJECT_PLACEHOLDER = ""
-_DOMAIN_PLACEHOLDER = ""
-_VERSION_PLACEHOLDER = ""
+_PROJECT_PLACEHOLDER = "{{ registration.project }}"
```

Comment:
Can you summarize why there aren't placeholders for auth and output prefix?

Reply:
neither are required during serialization

besides auth is a oneof - what should i have a placeholder for: the k8s service account or assumable iam role?

Reply:
done